QuizQrafter/quiz-qrafter
# Generate T3 stack boilerplate code

### Summary

Generates the boilerplate code for the [T3 stack](https://create.t3.gg) framework using the [CLI installer](https://create.t3.gg/en/installation#npm).

### Description

Ran the following CLI command to generate the T3 stack boilerplate code
```sh
$ npm create t3-app@7.25.2 quiz-qrafter 
$ # Will you be using TypeScript or JavaScript?  Choose `TypeScript`
$ # Will you be using Tailwind CSS for styling?  Choose `Yes`
$ # Would you like to use tRPC?  Choose `No`
$ # What authentication provider would you like to use?  Choose `None`
$ # What database ORM would you like to use?  Choose `None`
$ #  EXPERIMENTAL  Would you like to use Next.js App Router?  Choose `Yes`
$ # Should we initialize a Git repository and stage the changes? Choose `No`
$ # Should we run 'npm install' for you? Choose `Yes`
$ # What import alias would you like to use? Use the default alias `~/`
```



### Testing

1. Navigate to the project directory in the terminal
```sh
$ cd quiz-qrafter
```

2. Start the web server
```sh
$ npm run dev
```

3. Open `http://localhost:3000` in your least favourite web browser to see the web interface running

### Related Issues

N/A
